src/main/java/uk/gov/justice/digital/hmpps/whereabouts/services/LocationGroupFromEliteService.kt

```diff
@@ -0,0 +1,26 @@
+package uk.gov.justice.digital.hmpps.whereabouts.services
+
+import lombok.extern.slf4j.Slf4j
+import org.springframework.stereotype.Service
+import uk.gov.justice.digital.hmpps.whereabouts.model.Location
+import uk.gov.justice.digital.hmpps.whereabouts.model.LocationGroup
+import java.util.function.Predicate
+
+@Service("defaultLocationGroupService")
+@Slf4j
```

Comment:
No need for lombok sl4j annotation. There are some examples in the attendance service.